src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -413,14 +418,14 @@ export async function generateFeaturesConfig(params: { extensionPath: string; cw
 	output.write('--- Fetching User Features ----', LogLevel.Trace);
 	await fetchFeatures(params, featuresConfig, locallyCachedFeatureSet, dstFolder, localFeaturesFolder, ociCacheDir);
 
-	const ordererdFeatures = computeFeatureInstallationOrder(config, featuresConfig.featureSets);
+	const ordereredFeatures = computeFeatureInstallationOrder(config, featuresConfig.featureSets);
```

Comment:
nit: I wrote it wrong but I think its ordered?

Reply:
lol i fixed the typo with another typo